docs/proton.md

```diff
@@ -25,18 +25,18 @@
   <a href="#contributing"><strong>Contributing</strong></a> ·
   <a href="#need-help"><strong>Need help?</strong></a>
 </p>
-Proton is a streaming SQL engine, a fast and lightweight alternative to Apache Flink, 🚀 powered by ClickHouse. It helps data engineers and platform engineers solve real-time data pipelines and stream processing use cases, also powers the [Timeplus Cloud](https://timeplus.com) streaming analytics platform.
+Proton is a streaming SQL engine, a fast and lightweight alternative to Apache Flink, 🚀 powered by ClickHouse. It helps data engineers and platform engineers build data pipelines and solve problems in real-time, for various stream processing use cases. Proton also powers the [Timeplus Cloud](https://timeplus.com) streaming analytics platform.
```

Comment:
"solve problems in real-time" need refinement. I will change "Proton also.."

Reply:
Got it - main edit here is "solve data pipelines" doesn't sound correct, should be "build data pipelines"